assets/src/edit-story/components/checklist/checkCountContext.js

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import {
+  createContext,
+  useContext,
+  useEffect,
+  useState,
+  useCallback,
+} from 'react';
+import PropTypes from 'prop-types';
+
+export const CountContext = createContext(null);
+export const CategoryContext = createContext(null);
+
+const CHECK_CATEGORY = {
```

Comment:
nit: use the `ISSUE_TYPES` in checklist/constants? they're the same values. Could even update them to be `CHECK_CATEGORY` if that makes better sense semantically (it probably does).

Reply:
Oh good call. yea should definitely share that constant. I'm fine either way with regard to name. `ISSUE_TYPE` feels pretty synonymous with `CHECK_CATEGORY` so let me know if you have any preference, but fine with either

Reply:
0 pref 😹

Reply:
:hard-same: